lib/quantum/date_library.ex

```diff
@@ -14,4 +14,9 @@ defmodule Quantum.DateLibrary do
   Convert `NaiveDateTime` in UTC to `NaiveDateTime` in given tz.
   """
   @callback utc_to_tz(NaiveDateTime.t, String.t) :: NaiveDateTime.t | no_return
+
+  @doc """
+  Gives back the required application dependency to start, if any is needed.
+  """
+  @callback dependency_application :: atom | nil
```

Comment:
Hm, you're right is it....

I still think that we should mention it, so that the possibility of nil is very clear.